.github/workflows/build-scylla.yaml

```diff
@@ -14,6 +14,7 @@ on:
 
 jobs:
   build:
+    if: github.repository == 'scylladb/scylladb'
```

Comment:
Per [github docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository), this looks good. (I'd like to make this comment on this entire patch / commit, but github doesn't allow me to make comments at the commit level, only at the file/line level, as far as I can tell.)

... also, unrelated (again, can't comment on the commit as a whole): I suggest dropping "only" from the subject line. I propose `github: disable scheduled workflow on forks `

Reply:
resolved in v2

Reply:
... alas, I cannot "resolve this conversation", because I have no write access to this repo, nor am I the submitter of the PR: https://github.com/isaacs/github/issues/1952